base_img/init_functions

```diff
@@ -0,0 +1,418 @@
+# This file contains common functions used in init and in hooks
```

Comment:
Did you create all the code in this file, or was it copied from somewhere else?

Reply:
This code (along with almost everything else in base_img) was copied from mkinitcpio.